node/src/sub_lib/blockchain_bridge.rs

```diff
@@ -34,27 +36,33 @@ impl Debug for BlockchainBridgeSubs {
 }
 
 #[derive(Clone, PartialEq, Eq, Debug, Message)]
-pub struct ReportAccountsPayable {
+pub struct RequestAvailableBalancesForPayables {
     pub accounts: Vec<PayableAccount>,
     pub response_skeleton_opt: Option<ResponseSkeleton>,
 }
 
-impl SkeletonOptHolder for ReportAccountsPayable {
+impl SkeletonOptHolder for RequestAvailableBalancesForPayables {
     fn skeleton_opt(&self) -> Option<ResponseSkeleton> {
         self.response_skeleton_opt
     }
 }
 
 #[derive(Clone, PartialEq, Eq, Debug, Message)]
-pub struct SetDbPasswordMsg {
```

Comment:
I don't understand why this message was removed.

Reply:
Never used in the whole codebase. Technically dead code but masqueraded with the derive impls.

Reply:
That's a fault how `rustc` works. Thanks for cleaning it up.